swap_meet/item.py

```diff
@@ -1,2 +1,53 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+
+    def __init__(self, id=None, condition=0):
+        """
+        Wave 02: Represents a vendor object with a unique ID.
+        Attributes: id (int) and condition (float, default 0).
+        Method: get_category() returns a string with the class name.
+        """
+        if not id:
+            self.id = uuid.uuid4().int
+        else:
+            self.id = id
+        self.condition = condition
```

Comment:
We don't need `else` here. If the condition on line 11 doesn't evaluate to True and line 12 doesn't execute then Python will just continue running the rest of the code in this function. 

```suggestion
        self.id = id
        self.condition = condition
```